### PR TITLE
Fix db issue

### PR DESCRIPTION
**Pull request by nonococoleo**
# Pull Request Template

In this PR, update not commit bug has been fixed. As items are not duplicate in shopcarts, return value of find_item_by_sku_and_sid has been changed to the item object instead of a list of item objects, test cases also changed accordingly.

## Checklist:

- [x] I have written test cases for this change, and my code passed the test
- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -65,9 +65,8 @@ class Shopcart(db.Model):
     ##################################################
     id = db.Column(db.Integer, primary_key=True)
     user_id = db.Column(db.Integer)
-    create_time = db.Column(db.DateTime, nullable=False, server_default=str(datetime.utcnow()))
-    update_time = db.Column(db.DateTime, nullable=False, server_default=str(datetime.utcnow()),
-                            server_onupdate=str(datetime.utcnow()))
+    create_time = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    update_time = db.Column(db.DateTime, nullable=False, default=datetime.utcnow, onupdate=datetime.utcnow)
```
Comment (KHafner) [inline service/models.py:69]: This and line 187 are a bit too long so our pylint score dropped a little

**Review by KHafner — APPROVED** [review]
Great, thank you!

File: service/models.py (hunk 1 above)
Comment (nonococoleo) [reply service/models.py]: reformatted :)

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201118-151127 "deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201118-151127") to [dev](https://console.bluemix.net/apps/4aa42608-55fd-40d6-b27b-d99aa18a0658?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:dev"), including commit 029b897346c8e8ee2117811225776a3473452cfb

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028 "deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028") to [prod](https://console.bluemix.net/apps/ac06a6b3-040d-43d7-b290-6bfbba3f8c41?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:prod"), including commit 029b897346c8e8ee2117811225776a3473452cfb

**Pull request merged**